### DISCUSSION TITLE
How do you define 'duty'?

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10696] [Q&A]
Do you have a unit in OMEdit where I can input 'duty' for a heat exchanger? Which one is it?

**Comment by casella** [comment]
I guess the SI unit for a heat exchanger duty is W. You can then use multiple displayUnits, e.g. kW, MW, GW.

**Comment by ZackeyZAKI** [comment]
Thank you very much.

In other commercial software, there was a mention of "duty," but it was a field where you input the output values of each component.

I apologize for the confusion.

**Comment by casella** [comment]
Of course I thought 'duty' meant 'heat duty'. Maybe there are other meanings, to be honest I don't use commercial software for my modelling, because it's always weird stuff that need a custom approach.
